test/examples/Analytical/box_gamma_mixture.jl

```diff
@@ -8,25 +8,26 @@ include("../utils/plotting_helpers.jl")
 FT = Float64
 
```

Comment:
I know we are using SI now, but it's not clear to me whether we are working in kg (mass) or in m^3 (volume) as the particle attribute and the coalescence tensor, so I think we need to (a) choose which we want to use (I vote mass) and (b) keep commenting the SI units for clarity in all examples.